### PR TITLE
Fixes #1161 - Print message if there is nothing to backup

### LINKED ISSUE #1161: Print message that there's nothing to backup
From https://github.com/webcompat/webcompat.com/pull/1157

If you have an empty issues.db, there's nothing to backup. We should print something to that effect in the console.

### PR DESCRIPTION
**Pull request by MDTsai**
Issue #1161, follow-up of issue 1157, print message if there is nothing to backup
to BACKUP_DEFAULT_DEST

### CODE REVIEW

**Comment by miketaylr** [comment]
@MDTsai in a review, we typically ping someone for a review. For Python stuff, you can ask @karlcow or me (but Karl has better skills than me. ^_^).

r? @karlcow (or there's the new reviewers widget)

Drive by nit: Can you put a hash next to the issue number in the commit and PR title, so GitHub links them? i.e., #1157 and #1161?

File: run.py [hunk 1]
```diff
@@ -133,6 +133,8 @@ def config_validator():
             backup_files.sort()
             for old_file in backup_files[:-3]:
                 os.remove(app.config['BACKUP_DEFAULT_DEST'] + old_file)
+        else:
+            print ('There is nothing to backup to ' + app.config['BACKUP_DEFAULT_DEST'])
```
Comment (karlcow) [inline run.py]: * remove  the space in between print and the paranthesis.
* the line is tool long.


This is a way to do it.

```python
            print('There is nothing to backup to ' 
                  + app.config['BACKUP_DEFAULT_DEST'])
```

The commit message indeed needs a hashtag for the issue number.

Note: There's an opportunity to fix the syntax for this piece of code, the full code related to issues.db. It is ill-formed and needs to be fixed. @MDTsai if you want you can do it here. Or I will do it after in #1294 (as you wish).

Comment (MDTsai) [inline run.py]: @karlcow ok, I will submit another PR with #1294

**Comment by MDTsai** [comment]
@karlcow, I fix both #1161 and #1294 , please see if these two works.

File: run.py [hunk 2]
```diff
@@ -115,24 +115,35 @@ def config_validator():
...
                                            autoflush=False,
                                            bind=issue_engine)
         issue_db = scoped_session(issue_session_maker)
         # Take a backup if issues.db has data dump.
-        if issue_db().execute('select count(*) from webcompat_issues').fetchall()[0][0] > 0:
+        if (issue_db().execute('select count(*) from webcompat_issues')
+                      .fetchall()[0][0] > 0):
             if not os.path.exists(app.config['BACKUP_DEFAULT_DEST']):
-                print ('Creating backup directory at {}').format(os.path.join(app.config['BASE_DIR'], 'issues.db'))
+                print ('Creating backup directory at {}').format(
```
Comment (karlcow) [inline run.py]: no space between `print` and `(`

**Comment by karlcow** [comment]
Once these two minor things are fixed. :) We are good to go!

Thanks a lot @MDTsai and sorry for the delay. I had missed it. Make sure to do @karlcow when you want to specifically talk to me. :)

**Comment by karlcow** [comment]
which you did… I really missed it. Deep bow and apologizes.

**Comment by MDTsai** [comment]
Just remove the space between print and left brace, add another blank line after def config_validator(), the top def.

**Comment by karlcow** [comment]
Thanks @MDTsai 💯

**Comment by miketaylr** [comment]
+1 thanks @MDTsai!

**Pull request merged**